lib/Github/HttpClient/HttpClientInterface.php

```diff
@@ -100,6 +100,13 @@ public function setOption($name, $value);
     public function setHeaders(array $headers);
 
     /**
+     * Set base URL.
+     *
+     * @param string $baseUrl
+     */
+    public function setBaseUrl($baseUrl);
```

Comment:
and anyway, such change has nothing to do in the PR fixing the ManagementConsole. You should keep PRs focused on a single goal, otherwise it makes merging them much harder (as maintainers can only accept everything or nothing, even though some of the changes are worth it)